modules/kn-service-offline-about.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// serverless/cli_reference/kn-offline-services.adoc
+
+[id="kn-service-offline-about_{context}"]
+= About offline mode
+
+Normally, when you execute `kn service` commands, the changes immediately propagate to the cluster. However, as an alternative, you can execute `kn service` commands in offline mode:
+
+. When you create a service in offline mode, no changes happen on the cluster. Instead, the only thing that happens is the creation of the service descriptor file on your local machine.
+. After the descriptor file is created, you can manually modify it and track it in a version control system.
+// Once `update` works, add it here and make it into a list
+. Finally, you can propagate changes to the cluster. For that, you use commands such as `kn service create -f`, `kn service apply -f`, or `oc apply -f` on the descriptor files.
+
+The offline mode has several uses:
+
+* You can manually modify the descriptor file before using it to make changes on the cluster.
+* You can locally track the descriptor file of a service in a version control system. This enables you to reuse the descriptor file in places other than the target cluster, for example in Continuous Integration (CI) pipelines, development environments, or demos.
```

Comment:
```suggestion
* You can locally track the descriptor file of a service in a version control system. This enables you to reuse the descriptor file in places other than the target cluster, for example in continuous integration (CI) pipelines, development environments, or demos.
```

Not sure if this is the proper name of something, otherwise I think it should be lowercase?